typeDefinition/VideoChain.d.ts

```diff
@@ -1,3 +1,7 @@
 import {ElementChain} from './ElementChain';
+import {VideoStateModifiers} from './modifiers';
 
-export interface VideoChain extends ElementChain {}
+export interface VideoChain extends
+	ElementChain,
+	VideoStateModifiers<VideoChain>
```

Comment:
@JeKachan please review this changes

Reply:
If I understnad correctly VideoStateModifiers shoud be called once, so I think that  here should be ElementChain instead of VideoChain - VideoStateModifiers<ElementChain>

Reply:
You are right, thanks, but here is one more problem `video().toAssert().isPlaying()` throws error but `video().isPlaying().toAssert()` works. I could resolve it by moving  VideoStateModifiers to elementChain but in this case it will also be available for element(). Do you have any solution for this?

Reply:
I think it's OK to allow all those modifiers on element subject as well, because you can still define video as selector and it's up to IL to figure out if it's going to work e.g. `element({css: '#video'}).isPlaying()` - should be OK.

Reply:
At least this is how it works in UI test editor, so should be similar in JS API.

Reply:
updated